examples/feature_selection/select_from_model.py

```diff
@@ -0,0 +1,51 @@
+"""
+===================================================
+Feature selection using SelectFromModel and LassoCV
+===================================================
+
+Use SelectFromModel meta-transformer along with Lasso to select the best
+couple of features from the Boston dataset.
+"""
+# Author: Manoj Kumar <mks542@nyu.edu>
+# License: BSD 3 clause
+
+print(__doc__)
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from sklearn.datasets import load_boston
+from sklearn.feature_selection import SelectFromModel
+from sklearn.linear_model import LassoCV
+
+# Load the boston dataset.
+boston = load_boston()
+X, y = boston['data'], boston['target']
+
+# We use the base estimator since the L1 norm promotes sparsity of features.
+clf = LassoCV()
+
+# Set a minimum threshold of 0.25
+sfm = SelectFromModel(clf, threshold=0.25)
+sfm.fit(X, y)
+n_features = sfm.transform(X).shape[1]
+
+# Reset the threshold till the number of features equals two.
+# Note that the attribute can be set directly instead of repeatedly
+# fitting the metatransformer.
+while n_features > 2:
+	sfm.threshold += 0.1
+	X_transform = sfm.transform(X)
+	n_features = X_transform.shape[1]
+
```

Comment:
Should we also illustrate transform('mean')? Perhaps this belongs in narrative docs, which you are yet to commit.